src/v2i-hub/CARMACloudPlugin/src/CARMACloudPlugin.cpp

```diff
@@ -376,7 +377,7 @@ void CARMACloudPlugin::TCMAckCheckAndRebroadcastTCM()
 		}
 		else
 		{
-			PLOG(logDEBUG) << "NO TCMs to broadcast." << std::endl;
+			PLOG(logDEBUG4) << "NO TCMs to broadcast." << std::endl;
```

Comment:
Would it be useful to have a status that evaluates to the number of TCMs currently broadcasting rather than this log statement.

Reply:
Yeah, that might be a nice feature to add. This is an indicator that no more TCMs that is  left to broadcast.